real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -49,8 +51,8 @@
 	public static  String COM_PORT_IMU = "COM6";
 	public static  String COM_PORT_GPS_INTEGRATED = "COM7";
 	public static  String COM_PORT_GPS_STANDALONE = "COM8";
-	public static  String COM_PORT_ENCODER = "COM3";
-	public static  String COM_PORT_DRIVE_CONTROL = "COM9";
+	public static  String COM_PORT_ENCODER = "/dev/tty.usbmodem1411";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1743642470/issues/source?bid=2938665&fileBranchId=3058909#l54)